README.md

```diff
@@ -1 +1,45 @@
-# my-portfolio
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# My Portfolio
+
+Setting up a personal portfolio
+
+## Built With
+
+- html and css
+- linters
```

Comment:
- [ ] It would be better if you capitalize these lines in order to make your readme without grammar errors 😉.